integration/spark2/src/main/scala/org/apache/spark/sql/execution/datasources/CarbonFileFormat.scala

```diff
@@ -98,6 +97,8 @@ with Serializable {
       model,
       conf
     )
+    model.setTimestampformat(CarbonCommonConstants.CARBON_TIMESTAMP_DEFAULT_FORMAT)
```

Comment:
why not use value from CarbonProperties?

Reply:
It is fixed hive format we need to use. We already converted to this format in CarbonLoadDataCommand. Otherwise, spark cannot understand the data if we don't convert to this format